uaclient/version.py

```diff
@@ -54,12 +53,11 @@ def get_last_known_candidate() -> Optional[str]:
         not os.path.exists(CANDIDATE_CACHE_PATH)
         or os.stat(CANDIDATE_CACHE_PATH).st_mtime < last_apt_cache_update
     ):
-        candidate_version = None
         try:
-            policy = get_apt_cache_policy_for_package("ubuntu-advantage-tools")
-            match = re.search(CANDIDATE_REGEX, policy)
```

Comment:
I think we can remove `CANDIDATE_REGEX` defined above in this file since we don't use it here anymore